### ISSUE TITLE
My SSL experience

### ISSUE THREAD

**Issue by iamakimmer** [susanBuck/e28-fall20#53]
Just wanted to share my SSL woes in case anyone has a similar issue. My project uses the HTML5  audio recording feature but didn't realize today when I deployed to production that the audio recording doesn't work without HTTPS! It gets blocked. 

Fortunately, I went through the resource here: [https://hesweb.dev/e28/notes/production-server/https](https://hesweb.dev/e28/notes/production-server/https)

Restarted Apache, and the SSL page did work. However, now the browser stopped me connecting to the e28-api as that was still connecting with HTTP and complaining about mixed content. So I went through the same SSL creation steps. However, this time, it wasn't working. Digging ahead I noticed something went haywire by adding the second SSL entry:

![image](https://user-images.githubusercontent.com/335105/99355824-fbc25100-2876-11eb-90e5-1a1c1d846b1e.png)

This entry combined both my e28-api and p2 links? I'm not sure what happened, but I manually copied over another entry, restarted Apache, and a few hours later got it working.

**Comment by susanBuck** [comment]
Thanks for sharing these notes, @iamakimmer -

I'll be reviewing the SSL notes before I cover authentication, so I'll be on the lookout for this.

**Comment by susanBuck** [comment]
Following-up on this: encountered the same issue.

Works fine one the first domain, but goes wonky on the second. 

Glad you had posted this because it saved me from chasing my tail a bit when I encountered it. : )

We can manually fix it (like you did), but I'd like to get to the bottom of why it's not just working. I mention this in the videos I posted yesterday (Week 14) - I'll be punting the topic of https while I look further into this.
